ui/widgets/nuxeo-selectivity.js

```diff
@@ -6847,7 +6847,7 @@ typedArrayTags[weakMapTag] = false;
 
           .selectivity-load-more.highlight,
           .selectivity-result-item.highlight {
-            background: var(--nuxeo-primary-color);
+            background: var(--nuxeo-primary-color, #00aded);
```

Comment:
In the default theme,`#00aded` is used in `--nuxeo-action-color-activated`, while we are using `--nuxeo-primary-color: #0066ff;`. This fallback should use this last value, unless we have a clear design specification otherwise.
```suggestion
            background: var(--nuxeo-primary-color, #0066ff);
```

Reply:
ah sure! I think I mixed the 2 values! I'll use your suggestion. Thanks!